catalog/app/global-styles.tsx

```diff
@@ -39,7 +39,11 @@ function GlobalStyles() {
   return null
 }
 
-export default function WithGlobalStyles({ children }) {
+interface WithGlobalStylesProps {
```

Comment:
Also, there is already type for `{ children }`
```ts
type WithGlobalStylesProps = React.PropsWithChildren<{}>
```

Reply:
oh nice